spec/factories/transformation_mapping_item.rb

```diff
@@ -0,0 +1,3 @@
+FactoryGirl.define do
+  factory :transformation_mapping_item
```

Comment:
@bzwei You are using `TransformationMappingItem.new` in the spec, is this factory needed at this point?

Reply:
`spec/models/factory_girl_spec.rb` requires all models to have a factory defined.